storage/include/s3/client.hpp

```diff
@@ -0,0 +1,289 @@
+// Concord
+//
+// Copyright (c) 2020 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the
+// "License").  You may not use this product except in compliance with the
+// Apache 2.0 License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the subcomponent's license, as noted in the LICENSE
+// file.
+//
+// This convenience header combines different block implementations.
+
+#include <libs3.h>
+#include <cstring>
+#include <thread>
+#include "Logger.hpp"
+#include "assertUtils.hpp"
+#include "storage/db_interface.h"
+
+using namespace std;
+using namespace concordUtils;
```

Comment:
I think it might be better for users to not inject these namespaces in a header.

Reply:
done